app/src/state/ducks/contributions.js

```diff
@@ -222,9 +222,10 @@ export function getContributions(contributionSearchAttrs) {
       const response = await api.getContributions(contributionSearchAttrs);
       if (response.status === 200) {
         const contributions = await response.json();
-        const data = normalize(contributions.data, [schema.contribution]);
```

Comment:
@iankerriganharris will this impact our state negatively?

Reply:
That's what I'm not sure about. But also if this doesn't get fixed before the presentation it's probably fine.